index.js

```diff
@@ -22,4 +22,38 @@ addTaskButton.addEventListener("click", () => {
   });
 
   console.log(tasks);
+
+  taskTable = document.getElementById("task-table");
+  const TaskDetails = document.createElement("tr");
+  TaskDetails.className = "table-box-row";
+
+  for (let taskArrayIndex in tasks) {
+    for (
+      let taskObjectIndex = 0;
+      taskObjectIndex < Object.values(tasks[taskArrayIndex]).length;
+      taskObjectIndex++
+    ) {
+
+      TaskDetails.innerHTML =
+        `<td class="table-box-color-tag-job">` +
+        "" +
+        `</td>` +
+        `<td class="table-box-task" id="show-task-title">` +
+        Object.values(tasks[taskArrayIndex])[1] +
+        `</td>` +
+        `<td class="table-box-time" id="show-task-time">` +
+        Object.values(tasks[taskArrayIndex])[3].toString().match(/\d\d:\d\d:\d\d/) +
+        `</td>` +
+        `<td class="table-box-check-box">` +
+        `<input
+          class="table-box-check"
+          type="checkbox"
+          id="checkbox-done"
+          name="checkbox-done"
+        />` +
+        `</td>`;
+
+      taskTable.append(TaskDetails);
```

Comment:
Instead of manipulating (append) the dome on each loop. You could create the whole html as a string and manipulate (innerHTML) it once at the end?

Reply:
I thought it's better to use create element on top then append the inner html ,cause if I put everything to inner html it would makes some lines duplicate.(I'm not sure about grammer I used :D )

Reply:
There are many ways to do such things. I'm not sure about my grammar either.

Reply:
I try to do it in your way so I'm still thinking about how to do.